plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -63,6 +64,19 @@ export default class FsInstrumentation extends InstrumentationBase<FS> {
             );
           }
           for (const fName of CALLBACK_FUNCTIONS) {
+            if (fName === 'exists') {
+              // handling separately because of the inconsistent cb style:
+              // `exists` doesn't have error as the first argument, but the result
+              if (isWrapped(fs[fName])) {
```

Comment:
there is no need to duplicate this if. could be moved up before `if (fName === 'exists') {` once for all.

Reply:
Absolutely! Thanks!